src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IUsingStatement.cs

```diff
@@ -33,15 +33,18 @@ public static void M1()
             string expectedOperationTree = @"
 IUsingOperation (OperationKind.Using, Type: null) (Syntax: 'using (var  ... }')
   Resources: 
-    IVariableDeclarationsOperation (1 declarations) (OperationKind.VariableDeclarations, Type: null) (Syntax: 'var c = new C()')
-      IVariableDeclarationOperation (1 variables) (OperationKind.VariableDeclaration, Type: null) (Syntax: 'c = new C()')
-        Variables: Local_1: C c
-        Initializer: 
-          IVariableInitializerOperation (OperationKind.VariableInitializer, Type: null) (Syntax: '= new C()')
-            IObjectCreationOperation (Constructor: C..ctor()) (OperationKind.ObjectCreation, Type: C) (Syntax: 'new C()')
-              Arguments(0)
+    IVariableDeclarationGroupOperation (1 declarations) (OperationKind.VariableDeclarationGroup, Type: null) (Syntax: 'var c = new C()')
```

Comment:
>IVariableDeclarationGroupOperation (1 declarations) (OperationKind.VariableDeclarationGroup, Type: null) (Syntax: 'var c = new C()') [](start = 4, length = 132)

I think we shouldn't have this node in the tree for this scenario. Syntactically we have VariableDeclarationSyntax, which maps to IVariableDeclarationOperation